tests/test_models/test_engine/test_file_storage.py

```diff
@@ -118,6 +118,8 @@ def test_get(self):
                         user_id=user.id,
                         place_id=place.id,
                         text="AWESOME!")
+        review.save()
+        self.assertEqual(review, models.storage.get("Review", review.id))
```

Comment:
It might be better practice to put your results inside a variable, so it is easier to see what is being asserted